### DISCUSSION TITLE
Opening Balance

### DISCUSSION THREAD

**Discussion by adityagovindaraj** [ananthakumaran/paisa#136] [General]
Hi, how to I start off with the opening balances (not just assets, bank balances, but also investments, cost, and their asset classes, etc)?

Paisa looks fantastic! Kudos!

**Comment by ananthakumaran** [comment]
It depends on what you want from paisa, for some accounts like Bank Accounts, you can start with the current balance (check [equity account](https://paisa.fyi/reference/accounts/#equity)), but for most of the other accounts the purchase date and price matter for xirr and other calculation. I suggest getting a CSV file from the respective sites and then using the [Import](https://paisa.fyi/reference/import/) functionality. Initial import is one of the hardest parts of using Paisa. Feel free to ask follow-up questions for specific issues.
